components/PralineForm/index.js

```diff
@@ -95,8 +101,54 @@ export default function ProductForm() {
     event.target.elements[0].focus();
   }
 
+  async function handleEdit(event) {
+    event.preventDefault();
+    console.log("###### EDIT");
+
+    if (ingredients.length === 0) {
+      console.warning("No ingredient selected");
```

Comment:
Typo.
```suggestion
      console.warn("No ingredient selected");
```
It would probably be a good idea to let the user know as well. For example, you could display a message somewhere that informs the user about the ingredients being empty. Maybe something for an upcoming user story.

Reply:
Good hint, I will do a separate Userstory for error and success messages for the user.